### PR TITLE
add principles management to template

### PR DESCRIPTION
**Pull request by ajluk**
This Pr only adds Principles of management to template.

### CODE REVIEW

File: styles/books/pl-u-physics/book.scss [hunk 1]
```diff
@@ -9,9 +9,9 @@ $PagesWithBands: (
   $ChapterIntroType: fullWidth;
   
   @import 'framework/framework';
-  @import 'templates/template1/design';
-  @import '../../designs/carnival/pdf/locale/pl/folio';
-  @import 'templates/template1/settings';
+  @import 'design-settings/carnival/design';
+  @import 'design-settings/carnival/settings';
+  @import '../../designs/carnival/pdf/locale/pl/folio';
```
Comment (ajluk) [inline styles/books/pl-u-physics/book.scss:15]: @brittany-johnson I have updated the branch with `new-x.x` just to check if it resolve the problem with not passing checks and there were some paths that needed to be fixed after your changes, so I have done it within this PR.

**Review by Stackblocks — APPROVED** [review]

File: styles/output/intr-business-pdf.css [hunk 2]
```diff
@@ -0,0 +1,1971 @@
+@charset "UTF-8";
```
Comment (brittany-johnson) [inline styles/output/intr-business-pdf.css:1]: I noticed that two of the files for intro-business are spelled `intr-business`

**Review by brittany-johnson — CHANGES_REQUESTED** [review]
Just have one comment, other than that this is a great solution, good job!

File: styles/output/intr-business-pdf.css (hunk 2 above)
Comment (ajluk) [reply styles/output/intr-business-pdf.css:1]: Thank you for noticing! I missed it somehow.

**Review by brittany-johnson — APPROVED** [review]
Yay!!

**Review by Stackblocks — APPROVED** [review]

**Pull request merged**